examples/data-objects/clicker/src/index.tsx

```diff
@@ -29,8 +29,7 @@ export interface IClickerEvents extends IEvent {
 /**
  * Basic Clicker example using new interfaces and stock component classes.
  */
-// eslint-disable-next-line @typescript-eslint/ban-types
-export class Clicker extends DataObject<object, undefined, IClickerEvents> {
+export class Clicker extends DataObject<{E: IClickerEvents}> {
```

Comment:
this is a good example of only setting events, but being able to skip the other params